_pages/docs/3_Configuration.md

```diff
@@ -126,13 +126,6 @@ Or, if your `build.log` file is compressed (most probably!):
 
     bzcat build.log.bz2 | ct-ng extractconfig >.config
 
-The above commands will dump the configuration to `stdout`, so to rebuild a
-toolchain with this configuration, just redirect the output to the `.config`
-file:
-
-    ${CT_TARGET}-ct-ng.config >.config
-    ct-ng oldconfig
-
```

Comment:
I did changed the way the toolchain config is saved a while back in https://github.com/crosstool-ng/crosstool-ng/commit/5a1dc8d95

So I think technically the first example should be
```
bzcat /usr/share/${CT_TARGET}-ct-ng.config.bz2 >.config
```

For versions prior to 1.27.0
```
${CT_TARGET}-ct-ng.config >.config
```

For extracting it from the build.log
```
ct-ng extractconfig <build.log >.config
```

Probably also worth mentioning `ct-ng upgradeconfig` which will try and bring a config forward.